### PR TITLE
Update linting.md

### PR DESCRIPTION
**Pull request by orhovy**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/27498

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

### CODE REVIEW

File: docs/integrations/linting.md [hunk 1]
```diff
@@ -229,3 +233,31 @@ More info at: https://mypy.readthedocs.io/en/latest/index.html
 [Bandit](https://github.com/PyCQA/bandit) is a tool designed to find common security issues in Python code.
 
 We run `bandit` with a confidence level of HIGH. In the rare case that it reports a false positive, you can execlude the code by adding a comment of the sort: `# nosec`. See: https://github.com/PyCQA/bandit#exclusions .
+
+
+## XSOAR Linter
+
+This linter is a custom linter, based on pylint, which main purpose is to catch errors regarding demisto code standarts. The linter is activated using pylints load plugins ability. We run this linter with custom demisto error and warning messages only (all other messages are disabled). On rare occasions you may encounter a need to disable an error/warning returned from this linter. Do this by adding an inline comment of the sort on the line you want to disable the error:
```
Comment (bakatzir) [inline docs/integrations/linting.md:240]:
```suggestion
This is a custom linter, based on pylint, which main purpose is to catch errors regarding Cortex XSOAR code standards. The linter is activated using pylint load plugins ability. We run this linter with custom Cortex XSOAR error and warning messages only (all other messages are disabled). On rare occasions you may encounter a need to disable an error/warning returned from this linter. Do this by adding an inline comment of the sort on the line you want to disable the error:
```

File: docs/integrations/linting.md [hunk 2]
```diff
@@ -229,3 +233,31 @@ More info at: https://mypy.readthedocs.io/en/latest/index.html
...
+
+```
+# pylint: disable=<error-name>
+```
+
+For example:
+
+```python
+a, b = ... # pylint: disable=print-exists
```
Comment (bakatzir) [inline docs/integrations/linting.md:249]:
```suggestion
print('Success!') # pylint: disable=print-exists
```

**Review by bakatzir — APPROVED** [review]

File: docs/integrations/linting.md [hunk 3]
```diff
@@ -229,3 +233,31 @@ More info at: https://mypy.readthedocs.io/en/latest/index.html
...
+# pylint: disable=<error-name>
+```
+
+For example:
+
+```python
+print('Success!') # pylint: disable=print-exists
+```
+
+Is is also possible to `disable` and then `enable` a block of code. For example (taken from CommonServerPython.py):
```
Comment (amshamah419) [inline docs/integrations/linting.md:252]:
```suggestion
It is also possible to `disable` and then `enable` a block of code. The following example is taken from CommonServerPython.py:
```

File: docs/integrations/linting.md [hunk 4]
```diff
@@ -229,3 +233,31 @@ More info at: https://mypy.readthedocs.io/en/latest/index.html
 [Bandit](https://github.com/PyCQA/bandit) is a tool designed to find common security issues in Python code.
 
 We run `bandit` with a confidence level of HIGH. In the rare case that it reports a false positive, you can execlude the code by adding a comment of the sort: `# nosec`. See: https://github.com/PyCQA/bandit#exclusions .
+
+
+## XSOAR Linter
+
+This is a custom linter, based on pylint, which main purpose is to catch errors regarding Cortex XSOAR code standards. The linter is activated using pylint load plugins ability. We run this linter with custom Cortex XSOAR error and warning messages only (all other messages are disabled). On rare occasions you may encounter a need to disable an error/warning returned from this linter. Do this by adding an inline comment of the sort on the line you want to disable the error:
```
Comment (amshamah419) [inline docs/integrations/linting.md:240]:
```suggestion
This is a custom linter, based on pylint, whose main purpose is to catch errors regarding Cortex XSOAR code standards. The linter is activated using the pylint load plugins ability. We run this linter only with custom Cortex XSOAR error and warning messages (all other messages are disabled). On rare occasions, you may encounter a scenario in which you need to disable an error or warning message from being returned from by linter. To do this add an inline comment, as shown below, on the line you want to disable the error:
```

File: docs/integrations/linting.md [hunk 5]
```diff
@@ -229,3 +233,31 @@ More info at: https://mypy.readthedocs.io/en/latest/index.html
...
+```python
+# pylint: disable=sys-exit-exists
+if IS_PY3:
+    pass
+else:
+    sys.exit(1)
+# pylint: enable=sys-exit-exists
+```
+
+**Note**: pylint can take both the error name and error code when doing inline comment disables. It is best to use the name which is clearer to understand.
```
Comment (amshamah419) [inline docs/integrations/linting.md:263]:
```suggestion
**Note**: Pylint can take both the error name and error code when using an inline comment disable message. Please note, that it is best to use the name which is most clear to understand.
```

**Review by amshamah419 — APPROVED** [review]
Looks good, just batch commit the changes

**Pull request merged**